Tools/WinMLRunner/src/Run.cpp

```diff
@@ -605,6 +598,18 @@ int run(CommandLineArgs& args, Profiler<WINML_MODEL_TEST_PERF>& profiler) try
                                                                      deviceCreationLocationStringified);
                                 }
                             }
+
+                            if (SUCCEEDED(lastHr) && args.IsPerIterationCapture())
+                            {
+                                output.WritePerIterationPerformance(args, model.Name().c_str());
+                            }
+
+                            // Clear up bind, eval performance metrics after iteration
+                            if (args.IsPerformanceCapture() || args.IsPerIterationCapture())
```

Comment:
There is already a reset to the profiler @line 529. If a reset was added here, the profiler object that's passed by reference would be reset before the "run" method returns. It would break users who use WinMLRunner as a static library and would like the profiler object populated when passed into run.

Reply:
Sorry, this was a merge error.  Removing it.